app/Http/Controllers/Api/AuthController.php

```diff
@@ -30,6 +32,15 @@ public function store(Request $request)
 
         if (Auth::attempt($request->only('mobile_number', 'password'))) {
             $user = auth()->user();
+
+            DB::table('users_login_history')->insert([
+                'user_id' => $user->id,
+                'ip_address' => $request->server('REMOTE_ADDR'),
+                'device_used' => $request->headers->get('user_agent'),
+                'created_at' => Carbon::now()->format('Y-m-d H:i:s'),
+                'updated_at' => Carbon::now()->format('Y-m-d H:i:s')
+            ]);
+
```

Comment:
Let's put this on a event listener so we can run this in the background and to make the controller cleaner